### PR TITLE
Feat: Pronoun field for biodrop users

### LINKED ISSUE #8618: [FEATURE] Pronoun field for linkfree users
### Description

Edit: 

To be extra gender inclusive, we should allow people to add their pronouns in a separate field in the `manage your profile` page. `/account/manage/profile`

It should be a dropdown list to include popular pronouns which can be found: https://en.pronouns.page/pronouns

Why approved? Because people will try to mock pronouns with their custom pronouns if you can have custom (the reason why the approved list exists)

To understand what I mean you should have a look at [pronouns.page](https://en.pronouns.page/@xkrishguptaa)

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by TanyaLagodich**
# Closes #8618

## Changes proposed

Add a `<select />` element to the profile page with popular pronouns from the website https://en.pronouns.page/pronouns. It can also be saved on the server.

- [x] - Correct; marked as done
- [X] - Correct; marked as done
- [ ] - Not correct; marked as **not** done

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 # misc
 .DS_Store
+.idea/
```
Comment (eddiejaoude) [inline .gitignore:14]: Btw you should add this to your global git ignore, so it effects all your git projects 

https://www.youtube.com/watch?v=oH3MuD6WkZI

File: pages/api/account/manage/profile.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ export async function getProfileApi(username) {
   let getProfile = await Profile.findOne({ username });
 
   if (!getProfile) {
-    log.info(`peofile not found for username: ${username}`);
+    log.info(`profile not found for username: ${username}`);
```
Comment (eddiejaoude) [inline pages/api/account/manage/profile.js:55]: Thank you 👍

File: .gitignore (hunk 1 above)
Comment (TanyaLagodich) [reply .gitignore:14]: @eddiejaoude thank you very much! I'll add it. Should I delete this line from .gitignore?

File: pages/account/manage/profile.js [hunk 3]
```diff
@@ -218,7 +221,17 @@ export default function Profile({ BASE_URL, profile, fileExists }) {
...
-
+                    <div className="col-span-3 sm:col-span-4">
+                      <div className="mt-1">
+                        <Select
+                            name="pronoun"
+                            label="Pronouns"
+                            value={pronoun}
+                            options={pronouns}
+                            onChange={(e) => setPronoun(e.target.value)}
+                        />
```
Comment (eddiejaoude) [inline pages/account/manage/profile.js:232]: The indentation doesn't match project style, please check Prettier plugin

```suggestion
                        <Select
                          name="pronoun"
                          label="Pronouns"
                          value={pronoun}
                          options={pronouns}
                          onChange={(e) => setPronoun(e.target.value)}
                        />
```

File: .gitignore (hunk 1 above)
Comment (eddiejaoude) [reply .gitignore:14]: It is ok, it can stay for now 👍  Thank you though

**Review by eddiejaoude — APPROVED** [review]
Works great 👍 Thank you 🚀

**Pull request merged**